cmd/kvdpa-cli/kvdpa-cli.go

```diff
@@ -26,26 +25,14 @@ const deviceTemplate = ` - Name: {{ .Name }}
 func listAction(c *cli.Context) error {
 	var devs []vdpa.VdpaDevice
 	var err error
-	if c.String("mgmtdev") != "" {
-		var bus, name string
-		nameParts := strings.Split(c.String("mgmtdev"), "/")
-		if len(nameParts) == 1 {
-			name = nameParts[0]
-		} else if len(nameParts) == 2 {
-			bus = nameParts[0]
-			name = nameParts[1]
-		} else {
-			return fmt.Errorf("Invalid management device name %s", c.String("mgmtdev"))
-		}
-		devs, err = vdpa.GetVdpaDevicesByMgmtDev(bus, name)
-		if err != nil {
-			return err
-		}
+	var mgmtDev = c.String("mgmtdev")
+	if mgmtDev != "" {
+		devs, err = vdpa.GetVdpaDevicesByPciAddress(mgmtDev)
```

Comment:
Why do you assume it's PCI? I think this should call `listVdpaDevicesWithBusDevName` directly

Reply:
Yes, now I realize the poor decision to expose the public function `GetVdpaDevicesByPciAddress`, as agreed will make the cleanup in the next release of the library.
I should probably call `GetVdpaDevicesByMgmtDev(busName, devName string)`, `listVdpaDevicesWithBusDevName` is a package private function.